Index: vendor/api-platform/doctrine-orm/Extension/EagerLoadingExtension.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/api-platform/doctrine-orm/Extension/EagerLoadingExtension.php b/vendor/api-platform/doctrine-orm/Extension/EagerLoadingExtension.php
--- a/vendor/api-platform/doctrine-orm/Extension/EagerLoadingExtension.php	
+++ b/vendor/api-platform/doctrine-orm/Extension/EagerLoadingExtension.php	(date 1739837521200)
@@ -181,7 +181,7 @@
                 $associationAlias = $existingJoin->getAlias();
                 $isLeftJoin = Join::LEFT_JOIN === $existingJoin->getJoinType();
             } else {
-                $isNullable = $mapping['joinColumns'][0]['nullable'] ?? true;
+                $isNullable = $mapping->joinColumns[0]->nullable ?? true;
                 $isLeftJoin = false !== $wasLeftJoin || true === $isNullable;
                 $method = $isLeftJoin ? 'leftJoin' : 'innerJoin';
 
